pkg/reconciler/v1alpha1/revision/revision_test.go

```diff
@@ -497,9 +497,15 @@ func TestMarkRevReadyUponEndpointBecomesReady(t *testing.T) {
 		}
 	}
 
-	// Wait for events to be delivered.
-	if err := h.WaitForHooks(time.Second * 3); err != nil {
-		t.Error(err)
+	select {
+	case got := <-fakeRecorder.Events:
+		msg := "Revision becomes ready upon all resources being ready"
+		want := fmt.Sprintf("Normal RevisionReady %s", msg)
```

Comment:
there is 0 non-constness in the `want`, so why is it Sprintf, rather than just 
`const want = Normal Revi... being ready`

Reply:
Done.